lib/b2b/oauth_discovery.ts

```diff
@@ -0,0 +1,83 @@
+// !!!
+// WARNING: This file is autogenerated
+// Only modify code within MANUAL() sections
+// or your changes may be overwritten later!
+// !!!
+
+import { DiscoveredOrganization } from "./discovery";
+import { fetchConfig } from "../shared";
+import { request } from "../shared";
+
+// Request type for `Discovery.authenticate`.
```

Comment:
nit - should this be the full name of the method? `oauth.discovery.authenticate`

also, casing doesn't quite match since node is lowercase